release-notes/releases.json

```diff
@@ -37,7 +37,8 @@
         "checksums-sdk": "https://dotnetcli.blob.core.windows.net/dotnet/checksums/2.1.300-preview2-008533-sdk-sha.txt",
         "security": "FALSE",
         "lts-runtime": "FALSE",
-        "lts-sdk": "FALSE"
+        "lts-sdk": "FALSE",
+        "csharp-language": "7.3"
```

Comment:
Thanks for the heads-up.
Is `csharp-language` a well-known string? Should it be `csharplang`?
Should we also record VB language version?

I assume the .NET Core folks would be responsible for updating this when they pick up new versions of the compiler packages. Is that right?

Reply:
This is still hand authored and i just found out about it today. Im' trying to back fill all of the versions that I know of. 

The pattern seemed to be using `-` to separate words in this file hence I followed that for `csharp-language`.

Reply:
Who owns updating this?

Reply:
Working that out.

Reply:
@jaredpar Did we resolve the question of ownership? Thanks